src/code/components/modal-alert.js

```diff
@@ -1,65 +1,21 @@
 /*
  * Based on ReactOverlays demo at http://react-bootstrap.github.io/react-overlays/examples/#modals
  */
-import Modal from 'react-overlays/lib/Modal';
-import Button from './button';
-import ChallengeAwardView from './challenge-award';
+import EndLevelDialogView from '../fv-components/end-level-dialog';
 import React, { PropTypes } from 'react';
-import t from '../utilities/translate';
```

Comment:
Since `modal-alert` no longer displays user messages, a lot of its functionality can be removed. Really, these deletions belonged in the `hudNotifications` PR.